docs/12-design.md

```diff
@@ -55,23 +55,20 @@ sometimes be used to avoid reading a lot of the data).
 ## Tables
 
 All records in a table conform to a schema. The records in a table are stored in multiple files, with each file
-belonging to a partition. These files are all in an S3 bucket that is exclusively used by this table.
+belonging to a partition. These files are all stored in an S3 bucket deployed at an instance level, referred to as 
+the data bucket.
 
 Each table has a state store associated to it. This stores metadata about the table, namely the files that are in
 the table and how the records in the table are partitioned.
 
-Tables are deployed by the CDK table stack. This stack creates the infrastructure for each table. Each table
-requires a bucket where its data will be stored, and a state store. When a table is first created, its state store
-must be initialised.
+When an instance is deployed, it creates an S3 data bucket and creates the infrastructure for the state store. 
+When a table is created, a folder is created in the S3 data bucket for that table, and the state store is initialised 
+(to achieve this, both state store implementations support storing metadata for files in all tables). 
 
-To achieve this, the table stack obtains a list of the table properties files from the instance properties. For
-each table properties file, it creates the data bucket for that table and creates the state store. The creation
-of the state store is delegated to either the DynamoDB state store stack or the S3 state store stack, as appropriate.
-A custom CDK resource is then used to call `sleeper.cdk.custom.SleeperTableLambda`. This lambda initialises the
-state store and updates the table properties file which is stored in the instance's config bucket.
-
-The name of the bucket containing a table's data is called `sleeper-<instance-id>-table-tablename`, e.g.
-`sleeper-mySleeperInstance-table-table1`.
+Tables can also be taken offline and put online. Offline tables will be ignored when it comes to finding partitions 
+that need splitting and creating compaction jobs, however you are still able to ingest files to them, and perform 
+queries against them. When creating a new table, the boolean table property `sleeper.table.online` determines whether 
```

Comment:
It might read better to split the second sentence here into two rather than use "however".

"Offline tables will be ignored when it comes to finding partitions that need splitting and creating compaction jobs. You are still able to ingest files to them, and perform queries against them."